Tools/jit/_stencils.py

```diff
@@ -2,11 +2,15 @@
 
 import dataclasses
 import enum
-import sys
 import typing
 
 import _schema
 
+# Number of 32-bit words needed to store the bit mask of external symbols
+SYMBOL_MASK_SIZE: int = 4
+
+known_symbols: dict[str | None, int] = {}
```

Comment:
Let's try to get rid of this global cache, since I would ideally like the classes to be able to used multiple times in the same process without sharing state like this. I'd need to think it through more, but maybe it should be moved to the `_Target` class?

Reply:
let me see what I can place it...

Reply:
I put it in the Target class. Have a look.